english/torch/torch.md

```diff
@@ -2,4 +2,6 @@ torch.md
 
 Upon inspection the object has a small depression.
 When pressed an incandescent glow is emitted and the room comes into view
-.  [Return to the twisted passages](../zork/zork.md)
\ No newline at end of file
+.  
+[Return to the twisted passages](../zork/zork.md)
+[CALL UPON YOUR SPIRIT ANIMAL!](../spirit_animal/spirit-animal.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.